contrib/get-gmp

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
You might want to discourage most folks from using this unless they know that they need to. The version installed by their distribution  is probably the version that they want to use. All that is needed is a comment here.

Reply:
Done. Is that enough to discourage people? :-)